spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/AbstractBinder.java

```diff
@@ -133,82 +135,97 @@ public void setIntegrationEvaluationContext(EvaluationContext evaluationContext)
 
 	@Override
 	public final void afterPropertiesSet() throws Exception {
-		Assert.notNull(this.applicationContext, "The 'applicationContext' property must not be null");
+		Assert.notNull(this.applicationContext,
+				"The 'applicationContext' property must not be null");
 		if (this.evaluationContext == null) {
-			this.evaluationContext = ExpressionUtils.createStandardEvaluationContext(getBeanFactory());
+			this.evaluationContext = ExpressionUtils
```

Comment:
This is one example where I don't think we want to wrap. Generally speaking, I don't agree with forcibly wrapping lines at 80 chars.